termite.cc

```diff
@@ -1636,6 +1636,13 @@ static void on_alpha_screen_changed(GtkWindow *window, GdkScreen *, void *) {
     gtk_widget_set_visual(GTK_WIDGET(window), visual);
 }
 
+static void spawn_callback([[maybe_unused]] VteTerminal *terminal, GPid pid, GError *error, gpointer user_data) {
+    if (error) {
```

Comment:
The manual states that if the terminal is closed during the spawning process the callback will be called with `terminal == NULL`, it doesn't state anything regarding guarantees of `pid`and `error`. Might be safer to do this:
```suggestion
    if (!terminal) {
        // potentially free user_data, but it's nullptr in your code.
        return;
    }
    if (error) {
```